src/Braccio++.cpp

```diff
@@ -409,6 +409,7 @@ void BraccioClass::lvgl_init()
 
 void BraccioClass::display_thread_func()
 {
+  delay(200);
```

Comment:
Which underlying ressources would not be ready? What would be `lv_fs_read` be needed for? To my knowledge the LVGL filesystem is not used, the GIF for the splash screen is a binary array stored in flash. What's the callstack when `lv_fs_read` fails?